content/reference-rippled.md

```diff
@@ -2177,7 +2175,7 @@ An example of the request format:
 *Commandline*
 
 ```
-#Syntax account_tx account [ledger_index_min [ledger_index_max [limit]]] [binary] [count] [forward]
+#Syntax account_tx account [ledger_index_min [ledger_index_max [limit]]] [binary] [forward]
 rippled account_tx r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59 -1 -1 2 false false false
```

Comment:
If you don't revert the change from the previous line, then you need to change this to have only two "false"s.

Reply:
Reverted the change. Checking with Nik on whether count still exists for command line.